vars/mavenCanaryRelease.groovy

```diff
@@ -7,6 +7,9 @@ import org.apache.maven.model.Profile
 
 @Field final String FMP_STABLE_VERSION = "3.5.38"
```

Comment:
yes, @rupalibehera I think we should upgrade this also to FMP 3.5.40, so that it will also help us in https://github.com/openshiftio/openshift.io/issues/3058